.github/ISSUE_TEMPLATE/docs_question.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Documentation question/improvement
+about: Ask a question the documentation doesn't answer
+title: "[DOC]"
+labels: documentation
+assignees: ''
+---
+
+**What is currently unclear to you? Describe the problem you are experiencing.**
+E.g: I can't figure out how to [...]
```

Comment:
you can use comments instead `<!-- Comment -->`

Reply:
You mean to use comments for the example? Like this `<!-- For example: I can't figure out how to [...] -->` ?

If this is the case I should change it for the [bug report format](https://github.com/GmodNET/GmodDotNet/blob/main/.github/ISSUE_TEMPLATE/bug_report.md) as well

Reply:
Yeah